sys-apps/pcsc_lite/pcsc_lite-1.8.25.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="Middleware to access a smart card using SCard API (PC/SC)"
+DESCRIPTION="PSCS-lite implements the PC/SC international standard for PC \
+to smartcard reader communication."
+HOMEPAGE="https://pcsclite.apdu.fr"
+COPYRIGHT="1999-2002 David Corcoran
+	2001-2014 Ludovic Rousseau
+	2003-2004 Damien Sauveron"
+LICENSE="MIT
+	GNU GPL v2
+	BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://pcsclite.apdu.fr/files/pcsc-lite-$portVersion.tar.bz2"
+CHECKSUM_SHA256="d76d79edc31cf76e782b9f697420d3defbcc91778c3c650658086a1b748e8792"
+SOURCE_DIR="pcsc-lite-$portVersion"
+PATCHES="pcsc_lite-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="0.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	pcsc_lite$secondaryArchSuffix = $portVersion
+	lib:libpcsclite$secondaryArchSuffix = $libVersionCompat
+	lib:libpcscspy$secondaryArchSuffix = $libVersionCompat
+ 	cmd:pcsc_spy$commandSuffix = $portVersion
+	cmd:pcscd$commandSuffix = $portVersion
```

Comment:
cmd's before libs, also, there is an outstanding policy error if you build this for secondary architecture on gcc2h